### PR TITLE
Update _footer.html.erb

### PR DESCRIPTION
**Pull request by nycbeardo**
added RSS feed links to Update_footer

Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by alaxalves** [comment]
@nycbeardo Could this edit the template and link the issue you've solved and such? Thnx :smile:

**Comment by alaxalves** [comment]
Hey @nycbeardo I meant to link the issue like this: Fixes #434. Make sure you do this on your next MRs. Anyway this look ok to me ;)

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -8,8 +8,15 @@
       <p>
         Having trouble? <a href="http://github.com/publiclab/mapknitter/issues">Submit bugs here</a>.
       </p>
-        <p>
-          MapKnitter uses <a href="http://gdal.org">GDAL</a>, <a href="http://leafletjs.com">Leaflet</a>, <a href="http://rubyonrails.org">Ruby on Rails</a>, <a href="http://imagemagick.org">ImageMagick</a>, and other open source software, and makes extensive use of the open source font <a href="http://www.theleagueofmoveabletype.com/junction">Junction</a>.</p>
+         <p>
++       MapKnitter uses <a href="http://gdal.org">GDAL</a>, <a href="http://leafletjs.com">Leaflet</a>, <a href="http://rubyonrails.org">Ruby on Rails</a>, <a href="http://imagemagick.org">ImageMagick</a>, and other open source software, and makes extensive use of the open source font <a href="http://www.theleagueofmoveabletype.com/junction">Junction</a>.
```
Comment (alaxalves) [inline app/views/layouts/_footer.html.erb]: I don't like the idea of having this in one huge line, but let's wait on someone elses review.

**Comment by alaxalves** [comment]
Apparently there already has a merge solving this in #586. I don't know how to proceed. :smile: :smile: but it seems that #586 has been done before this one, maybe you could get another fto issue @nycbeardo ;)

**Comment by nycbeardo** [comment]
@alaxalves thank you for letting me know and the info you provided! I got nervous about this one

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (theghostyced) [reply app/views/layouts/_footer.html.erb]: Hi @alaxalves. What would be your preferred solution, because it does look good to me

Comment (alaxalves) [reply app/views/layouts/_footer.html.erb]: Just break the lines, that's all.

**Comment by sashadev-sky** [comment]
Hi @nycbeardo sorry that this issue had a PR open already. Would you like me to create a new one for you? If so just comment in our plots2 FTO tracker and say you're looking for a mapknitter issue: https://github.com/publiclab/plots2/issues/4963

Or you can get to work on one that is currently opened in another repo: https://github.com/publiclab/leaflet-environmental-layers/issues/177

Closing this PR due to duplication

**Pull request closed without merging**